### PR TITLE
[TEVA-1103] Job listing design changes

### PR DESCRIPTION
**Pull request by cpjmcquillan**
This PR implements the awesome new job listing page design.

## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-1103

## Changes in this PR
- Job banner link component
- Pill link component
- Awesome new job listing page design

### Before
Less awesome
image

### After
More awesome
image

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -343,7 +343,7 @@ en:
         other: '%{count} filters applied'
     notifications:
       content:
-        multi_school_job_notification_html: This teaching role is at <span class="govuk-!-font-weight-bold">more than one school in the trust</span>. You can find out more in Schools overview.
+        multi_school_job_notification_html: This teaching role is at <span class="govuk-!-font-weight-bold">more than one school in the trust</span>. You can find out more in <a class='govuk-link govuk-link--no-visited-state' href='#school-overview'>schools overview</a>.
```
Comment (david-mears-dfe) [inline config/locales/en.yml:346]: :)

**Review by david-mears-dfe — COMMENTED** [review]
I have left some questions and comments.

File: app/frontend/styles/components/jobs-banner.scss [hunk 2]
```diff
@@ -1,11 +1,62 @@
...
+  padding-bottom: govuk-spacing(3);
   padding-top: govuk-spacing(1);
   position: relative;
-  right: 50%;
   width: 100vw;
+
+  a.banner-link {
+    background-position: top 0 left 0;
+    background-repeat: no-repeat;
+    background-size: 20px 25px;
```
[the remark this answers is not in the document]
Comment (cpjmcquillan) [reply app/frontend/styles/components/jobs-banner.scss:14]: Background image is set depending on which class is added (no class added no background image set)

File: app/frontend/styles/components/jobs-banner.scss [hunk 3]
```diff
@@ -1,11 +1,62 @@
...
...2.1755 20.034C11.5745 20.6174 10.8494 20.9091 10 20.9091C9.15064 20.9091 8.42548 20.6174 7.82452 20.034C7.22356 19.4506 6.92308 18.7466 6.92308 17.9221H1.53846C1.12179 17.9221 0.761218 17.7743 0.456731 17.4787C0.152244 17.1831 0 16.8331 0 16.4286C0.400641 16.1019 0.765224 15.7596 1.09375 15.4018C1.42228 15.044 1.76282 14.5792 2.11538 14.0075C2.46795 13.4357 2.76643 12.8193 3.01082 12.1581C3.25521 11.4969 3.45553 10.6957 3.61178 9.75446C3.76803 8.81324 3.84615 7.80202 3.84615 6.72078C3.84615 5.53842 4.3149 4.43968 5.2524 3.42456C6.1899 2.40945 7.41987 1.79299 8.94231 1.57518C8.87821 1.42739 8.84615 1.2757 8.84615 1.12013C8.84615 0.808983 8.95833 0.544508 9.18269 0.326705C9.40705 0.108902 9.67949 0 10 0C10.3205 0 10.5929 0.108902 10.8173 0.326705C11.0417 0.544508 11.1538 0.808983 11.1538 1.12013C11.1538 1.2757 11.1218 1.42739 11.0577 1.57518C12.5801 1.79299 13.8101 2.40945 14.7476 3.42456C15.6851 4.43968 16.1538 5.53842 16.1538 6.72078C16.1538 7.80202 16.232 8.81324 16.3882 9.75446C16.5445 10.6957 16.7448 11.4969 16.9892 12.1581C17.2336 12.8193 17.5321 13.4357 17.8846 14.0075C18.2372 14.5792 18.5777 15.044 18.9062 15.4018C19.2348 15.7596 19.5994 16.1019 20 16.4286Z" ></path></svg>');
```
[the remark this answers is not in the document]
Comment (cpjmcquillan) [reply app/frontend/styles/components/jobs-banner.scss:42]: Made a ticket for deciding what we want to do and doing it https://dfedigital.atlassian.net/browse/TEVA-1316

File: app/views/shared/vacancy/_jobseeker_view.html.haml [hunk 4]
```diff
@@ -22,29 +22,34 @@
...
-      = render(Jobseekers::SchoolOverviewComponent.new(vacancy: @vacancy))
-      = render(Jobseekers::SchoolsOverviewComponent.new(vacancy: @vacancy))
-
-    %section
-      %h4.govuk-heading-s.share-this-job
-        = t('jobs.share_this_job')
-
+    .content-pills{ class: 'govuk-!-margin-bottom-5' }
+      %h3.govuk-heading-s{ class: 'govuk-!-margin-bottom-3' } Contents
+      .govuk-grid-row
```
[the remark this answers is not in the document]
Comment (cpjmcquillan) [reply app/views/shared/vacancy/_jobseeker_view.html.haml:27]: Indeed you are correct, I can add a comment

File: app/frontend/styles/components/jobs-banner.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (david-mears-dfe) [reply app/frontend/styles/components/jobs-banner.scss:42]: Many thanks were thanked that day.

**Review by david-mears-dfe — APPROVED** [review]

**Pull request merged**